src/app/services/canvas.service.ts

```diff
@@ -1296,6 +1330,8 @@ export class CanvasService {
                 }
             });
         }
+        // TODO: Should this return true here?
+        // If we get here the PUT/POST requests should have executed without an error.
```

Comment:
Thanks for catching this! Yes, we should return `true` here. The return value of this function is intended to indicate whether an individual-level override is created as specified, which is why it returns `false` if an existing inidvidual-level override is found for this student.